computer-core/src/main/java/com/baidu/hugegraph/computer/core/compute/input/EdgesInput.java

```diff
@@ -207,10 +213,20 @@ public Edge next() {
         }
     }
 
-    // TODO: use one reused Edges instance to read batches for each vertex.
+    /**
+     * Read edges & attach it by input stream, also limit the edges here
+     * TODO: use one reused Edges instance to read batches for each vertex.
+     */
     private Edges readEdges(RandomAccessInput in) {
         try {
+            // Limit edges to read here (unlimited by default)
```

Comment:
seems it's too later to limit the degree here

Reply:
add a todo for better way (later)